argo-workflows/sync-workflow.yaml

```diff
@@ -0,0 +1,29 @@
+apiVersion: argoproj.io/v1alpha1
```

Comment:
please put this in an openshift template, for [example](https://github.com/thoth-station/thoth-application/blob/master/adviser/base/openshift-templates/adviser.yaml)